src/user/entity/user.entity.ts

```diff
@@ -0,0 +1,38 @@
+import {
+    Column,
+    CreateDateColumn,
+    DeleteDateColumn,
+    Entity,
+    OneToMany,
+    PrimaryGeneratedColumn,
+    UpdateDateColumn,
+  } from 'typeorm';
+
+
+@Entity()
+export class User {
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @Column()
+    first_name: string;
```

Comment:
not first_name change name to firstName . and if you have firstName you dont need lastName ?